src/ua/lviv/iot/algo/part1/lab2/BusinessLaptop.java

```diff
@@ -0,0 +1,26 @@
+package ua.lviv.iot.algo.part1.lab2;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@ToString(callSuper = true)
+public class BusinessLaptop extends AbstractLaptop {
+    private int numberOfPorts;
+    private int warrantyInYears;
+
+    public BusinessLaptop(String model, float screenSize, int ram, int storage, int batteryLife, int batteryChargeInPercent, int numberOfPorts, int warrantyInYears) {
```

Comment:
Here I would definitely chop the line
```suggestion
    public BusinessLaptop(String model,
        float screenSize,
        int ram,
        int storage, 
        int batteryLife, 
        int batteryChargeInPercent, 
        int numberOfPorts, 
        int warrantyInYears
      ) {
```